Conch/conch.conf.dist

```diff
@@ -18,7 +14,14 @@
     # rollbar_environment => 'staging',
 
 	features => {
+		'audit' => 0,
 		#'orc' => 1,
+		'rollbar' => 1,
```

Comment:
Let's make the rollbar 0 in .dist, and uncomment orc and set as default off (or on, whichever you want).

Reply:
Oh, also mind running perltidy on this file. Mix of spaces and tabs

Reply:
Will do.

Reply:
Done.